sentiment/README.md

```diff
@@ -90,6 +90,17 @@ Other arguments of the `yarn train` command include:
   tensorboard (by default: http://localhost:6006) to view the loss and accuracy
   curves.
 
+  The links below point to hosted TensorBoard training los for various model
```

Comment:
I'd suggest switching Hosted TensorBoard to [TensorBoard.dev](https://tensorboard.dev). Also, "los" -> "loss"

Reply:
Thanks for the revew, @GalOshri . Done fixing the typo and changing the wording (with the CamelCasing you suggested).